### PR TITLE
Updated GwGm truth values and added workflow for benchmarks

### LINKED ISSUE #63: Run benchmarks as part of CI
### Desired capability or behavior.

We need to run benchmarks as part of CI.
To do all of them we'd have to install SNOPT.
I don't want to do this.
Let's make sure they run with IPOPT and add a workflow for it.

### Is your feature request related to a problem? Please describe.

_No response_

### Associated Bug Report

_No response_

### PR DESCRIPTION
**Pull request by johnjasa**
### Summary

Benchmark for GwGm fails on main.
I updated the values here and also introduced a workflow that runs the benches so regressions will be caught by CI in the future.

### Related Issues

- Resolves #63

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/validation_cases/benchmark_tests/test_FLOPS_based_sizing_N3CC.py [hunk 1]
```diff
@@ -274,7 +275,7 @@ def run_trajectory(sim=True):
         promotes_outputs=['mission:*'])
 
     traj.link_phases(["climb", "cruise", "descent"], [
-                     "time", Dynamic.Mission.ALTITUDE, Dynamic.Mission.MASS, Dynamic.Mission.DISTANCE], connected=strong_couple)
```
Comment (crecine) [inline aviary/validation_cases/benchmark_tests/test_FLOPS_based_sizing_N3CC.py:277]: why was altitude removed?

File: aviary/validation_cases/benchmark_tests/test_bench_GwGm.py [hunk 2]
```diff
@@ -13,26 +12,26 @@
...
         assert_near_equal(prob.get_val(Aircraft.Design.OPERATING_MASS),
-                          95509, tolerance=rtol)
+                          101555., tolerance=rtol)
 
         assert_near_equal(prob.get_val(Mission.Summary.TOTAL_FUEL_MASS),
-                          42529., tolerance=rtol)
+                          44098., tolerance=rtol)
 
         assert_near_equal(prob.get_val('landing.' + Mission.Landing.GROUND_DISTANCE),
-                          2634.8, tolerance=rtol)
```
Comment (crecine) [inline aviary/validation_cases/benchmark_tests/test_bench_GwGm.py:35]: where did these updated values come from?

File: aviary/validation_cases/benchmark_tests/test_FLOPS_based_sizing_N3CC.py (hunk 1 above)
Comment (johnjasa) [reply aviary/validation_cases/benchmark_tests/test_FLOPS_based_sizing_N3CC.py:277]: altitude is not a design variable here but is a constraint. prior versions of OM warned about this, recent ones throw an error.

File: aviary/validation_cases/benchmark_tests/test_bench_GwGm.py (hunk 2 above)
Comment (johnjasa) [reply aviary/validation_cases/benchmark_tests/test_bench_GwGm.py:35]: running the benchmark on main

**Comment by johnjasa** [comment]
Closing until we determine what caused the GwGm values to change

**Pull request closed without merging**